index.js

```diff
@@ -916,6 +916,34 @@ class Encore {
         return this;
     }
 
+    /**
+     * Configure the mini-css-extract-plugin.
+     *
+     * https://github.com/webpack-contrib/mini-css-extract-plugin#configuration
+     *
+     * ```
+     * Encore.configureMiniCssExtractPlugin(
+     *     function(loaderConfig) {
+     *         // change the loader's config
+     *         // loaderConfig.reloadAll = true;
+     *     },
+     *     function(pluginConfig) {
+     *         // change the plugin's config
+     *         // pluginConfig.chunkFilename = '[id].css';
+     *     }
+     * );
+     * ```
+     *
+     * @param {function} loaderOptionsCallback
+     * @param {function} pluginOptionsCallback
+     * @returns {Encore}
+     */
+    configureMiniCssExtractPlugin(loaderOptionsCallback, pluginOptionsCallback = () => {}) {
```

Comment:
should this be bundled in the webpack 5 upgrade, or be submitted separately ?

Reply:
I originally added it into #564 in case people wanted to disable the new `hmr` option (which is also included in this one). So, maybe we could ship it separately but it'd be preferable if it was done before the `mini-css-extract-plugin` upgrade.